packages/components/src/components/navigation-item/navigation-item.stories.ts

```diff
@@ -1,464 +1,161 @@
  */
 
-export const IndentedRelaxed = {
-  name: 'Indented × Relaxed',
-  parameters: {
-    controls: {
-      exclude: ['vertical', 'chevron', 'indented', 'sd-show', 'sd-hide']
-    }
-  },
-  render: (args: any) => {
-    return html`
-      ${generateTemplate({
-        axis: {
-          x: { type: 'attribute', name: 'indented' },
-          y: { type: 'attribute', name: 'relaxed' }
-        },
-        constants: [
-          { type: 'attribute', name: 'divider', value: true },
-          { type: 'attribute', name: 'current', value: true },
-          { type: 'attribute', name: 'vertical', value: true },
-          { type: 'attribute', name: 'chevron', value: true }
-        ],
-        args
-      })}
-    `;
-  }
+export const Chevron = {
+  render: () => html` <sd-navigation-item vertical chevron>Vertical Navigation with Chevron</sd-navigation-item>`
 };
 
 /**
- * Shows available slots.  `description`is only used when `vertical`is true.
+ * Add `children` slot to the navigation item to create an accordion.
+ * - Only works with `vertical` attribute.
+ * - A `chevron` will be added regardless of the `chevron` attribute.
+ * - The `open` attribute can be used to control the open state of the accordion.
  */
-
-export const Slots = {
-  parameters: {
-    controls: { exclude: [] }
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'description', 'children'].map(slot =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=...',
-              values: [
-                {
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--border slot--background h-8 w-[100px]"></div>`
-                      : `<div slot='${slot}' class="${cx(
-                          'slot slot--border slot--background h-6',
-                          slot === 'description' || slot === 'children' ? 'w-full' : 'w-6'
-                        )}"></div>`,
-                  title: slot
-                }
-              ]
-            }
-          },
-          constants: [
-            { type: 'template', name: 'width', value: '<div style="width: 300px">%TEMPLATE%</div>' },
-            {
-              type: 'attribute',
-              name: 'vertical',
-              value: true
-            },
-            { type: 'attribute', name: 'chevron', value: true },
-            {
-              type: 'attribute',
-              name: 'open',
-              value: true
-            },
-            defaultSlotConstant,
-            descriptionSlotConstant
-          ],
-          args: overrideArgs({ type: 'slot', name: 'default', value: '' }, args)
-        })
-      )}
-    `;
-  }
+export const Accordion = {
+  render: () =>
+    html`<sd-navigation-item vertical>
+      <div style="width: 245px; text-align: left;">Vertical Navigation with Accordion</div>
```

Comment:
The set width seems unnecessary ... there is enough space so it can run in 1 line